src/mbgl/gl/context.cpp

```diff
@@ -206,10 +207,12 @@ UniqueTexture Context::createUniqueTexture() {
     if (pooledTextures.empty()) {
         pooledTextures.resize(TextureMax);
         MBGL_CHECK_ERROR(glGenTextures(TextureMax, pooledTextures.data()));
+        stats.numCreatedTextures += TextureMax;
```

Comment:
could this statistics population affect the performance? shall we collect it conditionally, e.g. for debug builds only? @alexshalamov @astojilj

Reply:
Incrementing one integer value wont have any visible effect on performance :)

Reply:
true, but I mean the whole stats collecting in general, e.g. `std::chrono::steady_clock::now()` might not be that lightweight

Reply:
I think it's good idea to collect general rendering stats (in gl::Context). Frame time tracking in headless frontend could be collected conditionally, but impact on performance is neglectable nevertheless.